front/components/assistant/AssistantListActions.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  Button,
+  Chip,
+  ListAddIcon,
+  ListIcon,
+  ListRemoveIcon,
+} from "@dust-tt/sparkle";
+import type {
+  AgentUserListStatus,
+  LightAgentConfigurationType,
+  WorkspaceType,
+} from "@dust-tt/types";
+import React, { useContext, useState } from "react";
+
+import { SendNotificationsContext } from "@app/components/sparkle/Notification";
+import { updateAgentList } from "@app/lib/client/dust_api";
+
+type AssistantPreviewFlowType = "personal" | "workspace";
+
+interface AssistantListActions {
+  agentConfiguration: LightAgentConfigurationType;
+  owner: WorkspaceType;
+  isParentHovered: boolean;
+  flow: AssistantPreviewFlowType;
+}
+
+export default function AssistantListActions({
+  agentConfiguration,
+  isParentHovered,
+  owner,
+  flow,
+}: AssistantListActions) {
+  const { scope } = agentConfiguration;
+  const isGlobal = scope === "global";
+
+  if (isGlobal) {
+    return null; // Return null if isGlobal, since no actions are rendered in this case.
+  }
+
+  const [isHovered, setIsHovered] = useState(false);
+  const sendNotification = useContext(SendNotificationsContext);
+
+  // Function to determine if the assistant is added based on the flow and configuration.
+  const determineIfAdded = (
```

Comment:
a bit confusing; since we removed the logic of "add to workspace" from the gallery we can throw this part no?

Reply:
Yep, removed.